cmd/mqtt2prometheus.go

```diff
@@ -81,13 +86,24 @@ func main() {
 	}
 
 	mqtt_user := os.Getenv("MQTT2PROM_MQTT_USER")
-	mqtt_password := os.Getenv("MQTT2PROM_MQTT_PASSWORD")
-
 	if mqtt_user != "" {
 		cfg.MQTT.User = mqtt_user
 	}
-	if mqtt_password != "" {
-		cfg.MQTT.Password = mqtt_password
+
+	mqtt_password := os.Getenv("MQTT2PROM_MQTT_PASSWORD")
+	if *usePasswordFromFile {
+		if mqtt_password != "" {
```

Comment:
I think this should be:
```suggestion
		if mqtt_password == "" {
```
So in case of an empty password, throw an error.

Reply:
done!